### ISSUE TITLE
Image outputs darkening?

### ISSUE THREAD

**Issue by neosr-project** [2minkyulee/Noise-free-Optimization-in-Early-Training-Steps-for-Image-Super-Resolution#2]
Hi, thanks for your research. I'm trying to port ECO to [neosr](https://github.com/muslll/neosr) which uses the basicsr framework.
However, it looks like when alpha is >0.5, the outputs start to darken. Any idea why that would happen? Code:

```python
            self.output = self.net_g(self.lq)
            # ECO ("Empirical Centroid-oriented Optimization"):
            # https://arxiv.org/abs/2312.17526
            if self.eco and current_iter < self.eco_iters: # 200k total iters
                self.lq_output = F.interpolate(self.output, scale_factor=1/self.scale, mode="bicubic")
                a = min(current_iter / self.eco_iters, 1.0)
                self.lq = ((1 - a) * self.lq_output) + (a * self.lq)
                self.output = ((1 - a) * self.output) + (a * self.gt)
```
I have verified `self.lq` and `self.output` when `a=1.0` and they look correct (no issue, just the mixup of images), but during training it gets completely dark. The inputs (both LQ and GT) are normalized [0, 1] and float32. Here is the result when alpha == 0.9, followed by validation input:
![test](https://github.com/2minkyulee/Noise-free-Optimization-in-Early-Training-Steps-for-Image-Super-Resolution/assets/132400428/23f50110-6314-4621-8191-e0d9bc01c789)
![0887](https://github.com/2minkyulee/Noise-free-Optimization-in-Early-Training-Steps-for-Image-Super-Resolution/assets/132400428/e7b418d6-6fca-4851-a5b2-72ffccfa7a10)

**Comment by neosr-project** [comment]
I made a small mistake assigning the variables names. My bad.
